x11-misc/xscreensaver/xscreensaver-6.05.ebuild

```diff
@@ -7,7 +7,7 @@ inherit autotools flag-o-matic font optfeature pam strip-linguas
 
 DESCRIPTION="Modular screen saver and locker for the X Window System"
 HOMEPAGE="https://www.jwz.org/xscreensaver/"
-SRC_URI="https://www.jwz.org/xscreensaver/${P}.tar.gz"
+SRC_URI="https://www.jwz.org/xscreensaver/${P}.1.tar.gz"
```

Comment:
?

Why not just rename the ebuild?

Reply:
Upstream calls that version 6.05 names the file 6.05.1. (https://www.jwz.org/xscreensaver/download.html, see section Linux)

I think only having that in the SRC_URI is the least confusing way.

Reply:
> I think only having that in the SRC_URI is the least confusing way.
FWIW, I think so too. :)